zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/LocalSessionTracker.java

```diff
@@ -26,9 +28,11 @@
  * Local session tracker.
  */
 public class LocalSessionTracker extends SessionTrackerImpl {
+    ConcurrentMap<Long, Integer> sessionsWithTimeouts;
```

Comment:
Why not making the superclass' field `protected final` instead of `private final`?  Is this being made package-private on purpose?

Reply:
It was made package private because there was no need for sub classes of session tracker interface to access this. I agree for this patch it's better to change the accessor to protected which leads to less code.